### ISSUE TITLE
"Removing duplicate method calls from jTracert output" feature is broken in jTracert version 0.0.5

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ankit4u3/jtracert#13]
```
"Removing duplicate method calls from jTracert output" feature is broken in
jTracert version 0.0.5
```

Original issue reported on code.google.com by `Dmitry.Bedrin` on 4 Mar 2009 at 1:18

**Comment by GoogleCodeExporter** [comment]
```
Fixed in SVN revision r177
```

Original comment by `Dmitry.Bedrin` on 4 Mar 2009 at 1:35
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in jTracert version 0.1.0
```

Original comment by `Dmitry.Bedrin` on 17 Mar 2009 at 10:12
